src/pybind/mgr/orchestrator.py

```diff
@@ -635,6 +665,12 @@ def __init__(self, host_pattern, data_devices=None, db_devices=None, wal_devices
         #: A :class:`orchestrator.DeviceSelection`
         self.journal_devices = journal_devices
 
+        # TODO: is this actually implemented in DeepSea?
+        self.wal_size = wal_size
```

Comment:
Yes, it's called block_wal/db_size, to match the naming `ceph-volume` uses.

I'm open for discussion if we should just accept both versions.

Reply:
Hm, would you be ok with just using the names of ceph-volume here? this way we're not inventing a new name for the same thing.